site/content/docs/5.2/components/offcanvas.md

```diff
@@ -253,7 +253,7 @@ Since the offcanvas panel is conceptually a modal dialog, be sure to add `aria-l
 
 ### Variables
 
-<small class="d-inline-flex px-2 py-1 fw-semibold text-success bg-success bg-opacity-10 rounded-2">Added in v5.2.0</small>
+{{< added-in "5.2.0" >}}
```

Comment:
The Added in v5.2.0 has a green (#EAF9EA) background instead of the grey (#CCCCCC) one => green color to be replaced by the grey one.